star-dash/star-dash/ViewController.swift

```diff
@@ -37,17 +40,24 @@ class ViewController: UIViewController {
         self.renderer = renderer
     }
 
-    func setupGameEntities() {
+    private func createGameEngine() -> GameEngine {
+        let levelSize = self.storageManager?.getLevelSize(id: 0) ?? CGSize(width: 4_842, height: 1_040)
```

Comment:
I believe the id here should be consistent with method `setupGameEntities`, by passing a variable?